doc/classes/TileSetScenesCollectionSource.xml

```diff
@@ -56,38 +56,38 @@
 			<return type="bool" />
 			<param index="0" name="id" type="int" />
 			<description>
-				Returns whether this TileSet source has a scene tile with id [code]id[/code].
+				Returns whether this TileSet source has a scene tile with id [param id].
 			</description>
 		</method>
 		<method name="remove_scene_tile">
 			<return type="void" />
 			<param index="0" name="id" type="int" />
 			<description>
-				Remove the scene tile with id [code]id[/code].
+				Remove the scene tile with id [param id].
 			</description>
 		</method>
 		<method name="set_scene_tile_display_placeholder">
 			<return type="void" />
 			<param index="0" name="id" type="int" />
 			<param index="1" name="display_placeholder" type="bool" />
 			<description>
-				Sets whether or not the scene tile with id [code]id[/code] should display a placeholder in the editor. This might be useful for scenes that are not visible.
+				Sets whether or not the scene tile with id [param id] should display a placeholder in the editor. This might be useful for scenes that are not visible.
 			</description>
 		</method>
 		<method name="set_scene_tile_id">
 			<return type="void" />
 			<param index="0" name="id" type="int" />
 			<param index="1" name="new_id" type="int" />
 			<description>
-				Changes a scene tile's ID from [code]id[/code] to [code]new_id[/code]. This will fail if there is already a tile with a ID equal to [code]new_id[/code].
+				Changes a scene tile's ID from [param id] to [param new_id]. This will fail if there is already a tile with a ID equal to [param new_id].
 			</description>
 		</method>
 		<method name="set_scene_tile_scene">
 			<return type="void" />
 			<param index="0" name="id" type="int" />
 			<param index="1" name="packed_scene" type="PackedScene" />
 			<description>
-				Assigns a [PackedScene] resource to the scene tile with id [code]id[/code]. This will fail if the scene does not extend CanvasItem, as positioning properties are needed to place the scene on the TileMap.
+				Assigns a [PackedScene] resource to the scene tile with id [param id]. This will fail if the scene does not extend CanvasItem, as positioning properties are needed to place the scene on the TileMap.
```

Comment:
```suggestion
				Assigns a [PackedScene] resource to the scene tile with [param id]. This will fail if the scene does not extend CanvasItem, as positioning properties are needed to place the scene on the TileMap.
```